cli_flags.go

```diff
@@ -35,6 +37,24 @@ const (
 	maxArgMapScaleFactor = 8
 )
 
+type SymbolEndpoint struct {
+	Site   string
+	APIKey string
+	AppKey string
+}
+
+type SymbolEndpoints []SymbolEndpoint
+
+func (endpoints SymbolEndpoints) getFieldString(field string) []string {
+	data := make([]string, len(endpoints))
+	for _, e := range endpoints {
+		r := reflect.ValueOf(e)
+		f := reflect.Indirect(r).FieldByName(field)
+		data = append(data, f.String())
+	}
+	return data
+}
```

Comment:
Iterate on symbol endpoints and construct slices, instead of using `reflect` (will probably have also less performance overhead, since reflection is generally costly).